compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -565,6 +565,19 @@ class Allocator {
             storeSecondaryAllocationStackMaybe(Options, OldPtr, NewSize);
           }
         }
+
+        // If we've increased the size, fill the extra bytes.
+        if (NewSize > OldSize) {
+          const FillContentsMode FillContents =
+              TSDRegistry.getDisableMemInit() ? NoFill
+                                              : Options.getFillContentsMode();
+          if (FillContents != NoFill) {
```

Comment:
@ChiaHungDuan do you think we should add an `UNLIKELY` like the corresponding code in `initChunk()`?

Reply:
It's fine to leave it as it is. The UNLIKELY in initChunk may better to be removed as well but I'll check it later